save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt

```diff
@@ -33,6 +33,18 @@ class TestController {
         executionId?.let { testExecutionService.saveTestExecution(executionId, testsIds) }
     }
 
+    /**
+     * @param executionId ID of the [Execution], during which these tests will be executed
+     * @param testSuiteId ID of the [TestSuite], for which there will be created execution in DB
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    @PostMapping("/getAllTestsByTestSuiteIdAndSaveExecution")
+    fun getAllTestsByTestSuiteIdAndSaveExecution(@RequestBody executionId: Long, @RequestParam testSuiteId: Long) {
```

Comment:
```suggestion
    fun getAllTestsByTestSuiteIdAndSaveExecution(@RequestParam executionId: Long, @RequestParam testSuiteId: Long) {
```
Why not param?

Reply:
Because I'd like to pass to the uri only the `testSuiteId` and `executionId` by `BodyInserters` as more low level and static information